app/src/main/res/layout/fragment_management_data_settings.xml

```diff
@@ -0,0 +1,82 @@
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    app:title="@string/settingsDataManagementTitle">
+
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent">
+
+            <ImageView
+                android:id="@+id/dataManagementIllustration"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:importantForAccessibility="no"
+                android:src="@drawable/illustration_management_data_paper"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent" />
+        </androidx.constraintlayout.widget.ConstraintLayout>
+
+        <TextView
+            android:id="@+id/dataManagementTextDescription"
+            style="@style/Body"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:padding="@dimen/dividerHorizontalPadding"
+            android:text="@string/settingsDataManagementDescription"
+            app:layout_constraintTop_toBottomOf="@+id/settings_data_management_illustration" />
+
+        <com.google.android.material.button.MaterialButton
+            android:id="@+id/dataManagementSourceCodeButton"
+            style="@style/TextButtonSecondary"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="4dp"
+            android:text="@string/settingsDataManagementSourceCode"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/settings_data_management_text_description" />
+
+        <com.infomaniak.mail.ui.main.settings.ItemSettingView
+            android:id="@+id/dataManagementMatomo"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            app:icon="@drawable/ic_matomo"
+            app:itemAction="chevron"
+            app:title="@string/settingsMatomoTitle" />
+
+        <com.google.android.material.divider.MaterialDivider style="@style/ItemDivider" />
+
+        <com.infomaniak.mail.ui.main.settings.ItemSettingView
+            android:id="@+id/dataManagementSentry"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            app:icon="@drawable/ic_sentry"
+            app:itemAction="chevron"
+            app:title="@string/settingsSentryTitle" />
+    </LinearLayout>
+</com.infomaniak.mail.ui.main.menu.SimpleSettingView>
```

Comment:
There's probably a missing setting in your Android Studio that needs to be enabled to add empty lines at end of files.